features/steps/utils/ifc43x_alignment_validation/alignment.py

```diff
@@ -303,8 +303,9 @@ def _add_rep_curve_segments(
                         parent_curve_label += f"\nConstantTerm:{parent_curve.ConstantTerm}"
 
                     case "IfcPolynomialCurve":
+                        values = [f"{x:.4f}" for x in parent_curve.CoefficientsX]
```

Comment:
Didn't know it's possible to format numbers to strings with 4 decimals like that .. Thanks